projects/lxml/fuzz_xml_parse.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atheris
+import sys
+
+from lxml import etree as et
+
+def TestOneInput(data):
+  fdp = atheris.FuzzedDataProvider(data)
+
+  try:
+    root = et.XML(fdp.ConsumeUnicode(sys.maxsize))
+  except et.XMLSyntaxError:
+    None
+  return
+
+atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=True)
```

Comment:
hmm...maybe this should follow the more idiomatic way python code is written: 
```
def main():
  atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=True)
  atheris.Fuzz()


if __name__ == '__main__':
  main()
```

Reply:
Agreed - will also change this in the other python repositories I maintain.